docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -2,169 +2,83 @@
-The next input is the ``layers`` keyword which is ``"[SDO,AIA,304,1,100]"``.
-The first 4 are the observatory, instrument, detector, measurement values from before.
-Note that since SDO AIA has no detector value, you can skip this within the list.
-The ``1`` and ``100`` in the layer list refer to the visibility and opacity of the datasource.
-You can use the ``sourceid`` instead of the keywords, so it would be ``[13,1,100]`` for this example.
-
-Finally, the ``x0`` and ``y0`` are the center points about which to focus and the ``width`` and ``height`` are the pixel values for the image dimensions.
-These have defaults set so you do not need to supply these.
-
-In this example we will create a composite PNG image using data from two different SDO AIA wavelengths and LASCO C2 coronagraph data.
-The layer string is extended to include the additional data sources, and opacity is throttled down for the second AIA layer so that it does not completely block out the lower layer::
-
-   >>> from sunpy.net.helioviewer import HelioviewerClient
-   >>> import matplotlib.pyplot as plt
-   >>> from matplotlib.image import imread
-   >>> hv = HelioviewerClient()  # doctest: +REMOTE_DATA
-   >>> file = hv.download_png('2012/01/01', 6,
-   ...                        "[SDO,AIA,304,1,100],[SDO,AIA,193,1,50],[SOHO,LASCO,C2,white-light,1,100]",
-   ...                        x0=0, y0=0, width=768, height=768, watermark=True)  # doctest: +REMOTE_DATA
-   >>> im = imread(file)  # doctest: +SKIP
-   >>> plt.imshow(im)  # doctest: +SKIP
-   >>> plt.axis('off')  # doctest: +SKIP
-   >>> plt.show()  # doctest: +SKIP
-
-.. image:: helioviewer-4.png
-
-For more information about using querying Helioviewer.org, see the `Helioviewer.org
-API documentation <https://api.helioviewer.org/docs/v2/>`_.
+The Helioviewer Project now maintains a Python wrapper around their API.
+It is called `hvpy <https://hvpy.readthedocs.io/en/latest/>`__ and their documentation has several examples of how to use it.
+
+Migration guide
+===============
+
+If you are using ``HelioviewerClient`` this guide shows to switch to using ``hvpy``.
+
+``HelioviewerClient().data_sources`` is replaced by `hvpy.DataSource`, an `~enum.Enum` that lists every known data source for Helioviewer.org.
+
+.. code-block:: python
+
+   >>> import hvpy
+   >>> print(hvpy.DataSource.AIA_131)
+   DataSource.AIA_131
+
+It also supports tab-complete, to find the data source you want.
+
+``HelioviewerClient().get_data_sources`` is replaced by :func:`hvpy.getDataSources`.
+
+.. code-block:: python
+
+   >>> import hvpy
+   >>> hvpy.getDataSources()  # doctest: +REMOTE_DATA
+   {'SDO': {'HMI': {'continuum': {'sourceId': 18,
+   ...
+
+``HelioviewerClient().get_closest_image`` is replaced by :func:`hvpy.getClosestImage`.
+
+.. code-block:: python
+
+   >>> from datetime import datetime
+   >>> import hvpy
+   >>> hvpy.getClosestImage(date=datetime.now(), sourceId=hvpy.DataSource.AIA_171)  # doctest: +REMOTE_DATA
```

Comment:
```suggestion
   >>> hvpy.getClosestImage(date=datetime(2022, 1, 1), sourceId=hvpy.DataSource.AIA_171)  # doctest: +REMOTE_DATA
```
To stop the file returned changing depending on the date it was run 😄 . The return will need changing below too to match whatever date is put here.

Reply:
And it might be nice to include the full output below?

Reply:
Added